components/Footer/Footer.tsx

```diff
@@ -0,0 +1,79 @@
+import Link from "next/link"
+import React from "react"
+import styles from "./footer.module.css"
+
+const Footer = () => {
+	return (
+		<footer className={styles.footer}>
+			<div className={styles.footer__top}>
+				<div className={styles.footer__box}>
+					<h2 className={styles.footer__linkHeading}>Community</h2>
+					<ul className={styles.footer__linkList}>
+						<li className={styles.footer__linkItems}>
+							<a
+								href="https://github.com/AccessibleForAll"
+								className={styles.footer__links}>
+								Github Page
+							</a>
+						</li>
+						<li className={styles.footer__linkItems}>
+							<a
+								href="https://github.com/AccessibleForAll/AccessibleWebDev/blob/main/CONTRIBUTING.md"
+								className={styles.footer__links}>
+								Become a Contributor
+							</a>
+						</li>
+						<li className={styles.footer__linkItems}>
+							<a href="#" className={styles.footer__links}>
+								Sponsor Us
+							</a>
+						</li>
+					</ul>
+				</div>
+				<div className={styles.footer__box}>
+					<h2 className={styles.footer__linkHeading}>Company</h2>
+					<ul className={styles.footer__linkList}>
+						<li className={styles.footer__linkItems}>
+							<Link href="/">
+								<a className={styles.footer__links}>Home</a>
+							</Link>
+						</li>
+						<li className={styles.footer__linkItems}>
+							<Link href="/about">
+								<a className={styles.footer__links}>About</a>
+							</Link>
+						</li>
+						<li className={styles.footer__linkItems}>
+							<Link href="/about">
```

Comment:
This one can also redirect to the homepage for now
```suggestion
							<Link href="/">
```